Source/swiftlint/Commands/RulesCommand.swift

```diff
@@ -83,16 +83,21 @@ extension TextTable {
         ]
         self.init(columns: columns)
         let sortedRules = ruleList.list.sort { $0.0 < $1.0 }
-        for (ruleId, ruleType) in sortedRules {
+        for (ruleID, ruleType) in sortedRules {
             let rule = ruleType.init()
-            let configuredRule = configuration.rules.indexOf({
-                $0.dynamicType.description.identifier == ruleId
-            }).map { configuration.rules[$0] }
-            addRow(ruleId,
-                   (rule is OptInRule) ? "yes" : "no",
-                   (rule is CorrectableRule) ? "yes" : "no",
-                   configuredRule != nil ? "yes" : "no",
-                   ((configuredRule ?? rule)  as? _ConfigProviderRule)?.configDescription ?? "N/A")
+            let configuredRule: Rule? = {
```

Comment:
I'm not a fan of the current `indexOf(...).map { rules[$0] }` approach (get index of element matching a predicate, then subscript at that index to get the object again). This alternative is more verbose but feels more direct to me.